sound/effect.h

```diff
@@ -213,7 +213,7 @@ class Effect {
 	isDigit(rest[2]) &&
 	isDigit(rest[3])) {
       int sub = strtol(rest+1, nullptr, 10);
-      sub_files_ = std::max<int>(sub_files_, sub + 1);
+      sub_files_ = std::max<int>(sub_files_, sub);
```

Comment:
This is wrong.
Sub files now start at 000
so if you have 000, 001 and 002 you have 3 files.
002 + 1 == 3

Reply:
ok, so I needed to change in my font then, I thought it was only the alt that changed.  will test again.

Reply:
maybe sub_files_ shouldn't have 0 index?  unable to to Select(0) now, send PM in theCrucible with more detail.